CHANGELOG.md

```diff
@@ -65,6 +65,26 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+#### Make sure any individually imported JavaScript modules work as expected
+
+We've changed the naming of our component JavaScript modules so that individual imports are now attached to
+`window.GOVUKFrontend.[ComponentName]` instead of `window.GOVUKFrontend`.
+
+This means you can now import multiple modules without overwriting the previous one, for example:
+
+```
+//= require govuk/components/accordion/accordion.js
+//= require govuk/components/button/button.js
+
+# The above are available under window.GOVUKFrontend.Accordion and window.GOVUKFrontend.Button respectively
+```
+
+If you are importing indvidual component JavaScript, you should check any references to `window.GOVUKFrontend` in your code and update them to point to the correct component: `window.GOVUKFrontend.[ComponentName]`. You may have implemented workarounds that can now be removed.
```

Comment:
```suggestion
If you're importing individual component JavaScript, you should check any references to `window.GOVUKFrontend` in your code and update them to point to the correct component: `window.GOVUKFrontend.[ComponentName]`. You can now remove any workarounds you may have implemented.
```
Wonder if we could change "If you're importing individual component JavaScript" to "If you're importing JavaScript modules individually" (to somewhat mirror wording in heading title)? 
Also, should we tell users "you can" or "you should" now remove any workarounds?